jinja2/includes/icons/general/money-bill-alt.svg

```diff
@@ -0,0 +1,3 @@
+<svg class="icon icon-money-bill" xmlns="http://www.w3.org/2000/svg" width="23px" height="16px" viewBox="0 0 23 16" aria-hidden="true">
```

Comment:
We should `role="img"` to all of these SVG files.

Reply:
Addressed.